tests/testthat/test-update-with-missing.R

```diff
@@ -142,10 +142,8 @@ with_test_authentication({
             ),
             name = "All B"
         )
-        expect_equal(ids(categories(ds$cat)), 1)
-        ds$cat[5] <- NA
         expect_equal(ids(categories(ds$cat)), c(1, -1))
-        expect_equal(as.vector(ds$cat[3:7], mode = "id"), c(1, 1, -1, 1, 1))
+        expect_equal(as.vector(ds$cat[3:7], mode = "id"), c(1, 1, 1, 1, 1))
```

Comment:
Consider deleting these tests entirely. Presumably you have tests on the server side where variables are created without No Data explicitly and it gets added automatically, so these would be redundant. They were testing behavior on the R side to make sure that No Data is added when `NA` is set, and that's not being exercised anymore.

Reply:
☝️